### PR TITLE
Add Python quickstart

### PR DESCRIPTION
**Pull request by gvdongen**
Add Python quickstart

### CODE REVIEW

File: docs/get_started/quickstart.mdx [hunk 1]
```diff
@@ -532,6 +532,136 @@ go build .
...
+            ```shell
+            python3 -m venv .venv
+            source .venv/bin/activate
+            pip install -r requirements.txt
+            ```
+
+            Now, start developing your service in `example.py`. Run it with a [Hypercorn](https://pypi.org/project/Hypercorn/) server, and let it listen on port `9080` for requests:
+
+            ```shell
+            python -m hypercorn --config hypercorn-config.toml -b localhost:9080 example:app
```
Comment (slinkydeveloper) [inline docs/get_started/quickstart.mdx:579]: I don't think you need to specify the bind address here, it's in the config: https://github.com/restatedev/examples/blob/main/templates/python/hypercorn-config.toml

File: docs/get_started/quickstart.mdx [hunk 2]
```diff
@@ -532,6 +532,136 @@ go build .
...
+                ```shell CLI
+                # link(1:3) /develop/local_dev#running-restate-server--cli-locally
+                restate example python-hello-world &&
+                cd python-hello-world
+                ```
+
+                ```shell npx
+                npx @restatedev/restate example python-hello-world &&
+                cd python-hello-world
+                ```
```
Comment (slinkydeveloper) [inline docs/get_started/quickstart.mdx:555]: Showing npx is very weird here, i'm a python user i know nothing about npm :)

File: docs/get_started/quickstart.mdx [hunk 3]
```diff
@@ -532,6 +532,136 @@ go build .
...
+
+        </Step>
+
+        <Step stepLabel="3" title="Launch Restate">
+            Restate is a single self-contained binary. No external dependencies needed. Run it locally via ([or download the binaries](/develop/local_dev#running-restate-server--cli-locally)):
+
+
+            <CH.Code rows={"2"}>
+
+                ```shell npx
```
Comment (slinkydeveloper) [inline docs/get_started/quickstart.mdx:590]: Same comment as before, in particular here it's the first choice!

Comment (slinkydeveloper) [reply docs/get_started/quickstart.mdx:590]: Actually this comment applies to the other languages too, in java, golang, i get npx as first choice. it's ok if you wanna keep it, but it shouldn't be the first choice!

File: docs/get_started/quickstart.mdx [hunk 4]
```diff
@@ -532,6 +532,136 @@ go build .
...
+                ```
+
+            </details>
+
+        </Step>
+
+        <Step stepLabel="5" title="Send a request to the Greeter service">
+
+            ```shell
+            curl localhost:8080/Greeter/greet -H 'content-type: application/json' -d '"Hi"'
```
Comment (slinkydeveloper) [inline docs/get_started/quickstart.mdx:656]: We don't use yet `--json`? :)

File: docs/get_started/quickstart.mdx (hunk 2 above)
Comment (gvdongen) [reply docs/get_started/quickstart.mdx:555]: I show it everywhere, it's up for the user to decide on what he has installed...

File: docs/get_started/quickstart.mdx (hunk 4 above)
Comment (gvdongen) [reply docs/get_started/quickstart.mdx:656]: As I mentioned before, not until it's the default on linux installations...

File: docs/get_started/quickstart.mdx (hunk 3 above)
Comment (gvdongen) [reply docs/get_started/quickstart.mdx:590]: So do I then link to the download script for the binary or what should I put first? Cause I also don't like Docker as first option. And Homebrew is not general enough.

Comment (slinkydeveloper) [reply docs/get_started/quickstart.mdx:590]: download script for the binary is fine, but also docker i think it's fine.

File: docs/get_started/quickstart.mdx (hunk 4 above)
Comment (slinkydeveloper) [reply docs/get_started/quickstart.mdx:656]: isn't it already these days? that's from a quite old `curl` version right? (just checked, curl 7.82, 2.4 years ago :smile: )

File: docs/get_started/quickstart.mdx (hunk 3 above)
Comment (slinkydeveloper) [reply docs/get_started/quickstart.mdx:590]: But definitely IMO it's not fine that npx is the first choice for anything other than typescript

**Pull request closed without merging**